web/js/modules/core/actions.js

```diff
@@ -1,38 +1,66 @@
-export function requestAction(dispatch, actionName, url, mimeType, signal) {
-  dispatch(startRequest(actionName));
+export function requestAction(
+  dispatch,
+  actionName,
+  url,
+  mimeType,
+  id,
+  signal,
+  TIMEOUT_AMOUNT
+) {
+  // let didTimeOut = false;
```

Comment:
Remove commented out timeout functions if not planned for future use

Reply:
I left this one if since I though it might be useful in the future but if you all think I should take it out I will.